kiwi/bootloader/config/grub2.py

```diff
@@ -271,12 +270,14 @@ def setup_disk_boot_images(self, boot_uuid, lookup_path=None):
             by the distribution. The bios core image is created
             when grub2-install is called
         """
-        log.info('Creating grub bootloader images')
+        self.__copy_theme_data_to_boot_directory(lookup_path)
 
-        if self.firmware.efi_mode():
-            self.efi_boot_path = self.create_efi_path()
+        if not self.firmware.efi_mode():
+            return
 
-        self.__copy_theme_data_to_boot_directory(lookup_path)
+        log.info('Creating grub bootloader images')
+
+        self.efi_boot_path = self.create_efi_path()
 
```

Comment:
ok so with the return you want to skip the creation of the efi image which is architecture specific, got that. But with ppc we set firmware to 'ofw' which means the efi part is not touched. I agree we have to make the creation of the efi module architecture dependant (for e.g arm this has to change) but for the ppc support there is no need to touch it now or am I mistaken ?

I'd like to keep this changeset as clear and small and ppc focused as possible. This is because when I look at the old kiwi code things are so weird and unclear that I'd like to avoid it this time. Thanks